src/main/resources/static/PIPlayerScripts/EighthSessionSentencesNT.js

```diff
@@ -50,7 +50,34 @@ define([], function() {
             ]
```

Comment:
Diheng, this looks like a GIT conflict that was not resolved.  I don't think we want to add this in.